src/segmantic/seg/monai_unet.py

```diff
@@ -517,6 +631,7 @@ def print_table(header, vals, indent="\t"):
                 dice_np = dice.cpu().numpy()
                 print("Mean Dice: ", np.mean(dice_np))
                 print("Class Dice:")
+                all_mean_dice.append(dice_metric.aggregate().item())
```

Comment:
slightly unfortunate formatting. the lines
```
print("Class Dice:")
print_table(tissue_names, np.squeeze(dice_np))
```
belong together. please just move the line `all_mean_dice.append(dice_metric.aggregate().item())` above or below

Reply:
ahh ok~~ sorry for that